src/entities/trade.ts

```diff
@@ -576,4 +578,96 @@ export class Trade {
     }
     return methodName
   }
+
+  /**
+   * return the mistX router method name for the trade
+   * @param tradeType the type of trade, TradeType
+   * @param etherIn the input currency is ether
+   * @param etherOut the output currency is ether
+   * @param useFeeOnTransfer Whether any of the tokens in the path are fee on transfer tokens, TradeOptions.feeOnTransfer
+   * @param enforceUseFeeOnTransfer use to throw an invariant if there is no useFeeOnTransfer option for TradeType.EXACT_OUTPUT trades
+   */
+   public static estimateBribes(
+    pairs: Pair[],
+    currencyIn: Currency,
+    currencyOut: Currency, 
+    gasPriceToBeat: BigintIsh,
+    minerBribeMargin: BigintIsh,
+    { maxHops = 3 }: BestTradeOptions = {}
+  ): BribeEstimation | undefined {
+    const etherIn = currencyIn === ETHER
+    const etherOut = currencyOut === ETHER
+
+    const exactInGas = estimatedGasForMethod(Trade.methodNameForTradeType(TradeType.EXACT_INPUT, etherIn, etherOut), maxHops.toString())j
```

Comment:
`estimatedGasForMethod` in the SDK still has `const factor = parseBigintIsh('0') // TODO: change this` I guess we should re-visit that factor or keep it in mind

Reply:
We still need to figure out what that factor is, it will be a constant. Ill make a task to do this before MVP